### PR TITLE
Add new login entry view

### PR DESCRIPTION
**Pull request by fespinoza**
**Update**: the design is confirmed with thuy and the test was added. This PR is ready, but i would wait to merge after I work on the integration too, to ensure there is no unforeseen extra view requirements

# Why?

New design and text that better explain why the user should log in on each of the pages you are asked to log in.

# What?

Added a new full screen page and its demo view. It's my first PR doing an addition to the project so please check if I'm following the conventions correctly and so on.

I made sure the screen looks great on iPad too.

# Show me

**Note**: the "before" screen is not in FinniversKit

| Before | After |
| --- | :---: |
| IMG_4173 | IMG_4172 |

# Extra

- ~~I'm not sure about the 2: login and register buttons, as SPiD has a single UI for both flows. I need to double check with Thuy.~~
- ~~I will add a screenshot test for this view too, once I make sure the final design of the UI.~~
- ~~Even though this is a "WIP" PR, i would like a review for it.~~

### CODE REVIEW

File: Demo/Fullscreen/LoginEntry/LoginEntryViewDemoView.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import FinniversKit
+
+public class LoginEntryViewDemoView: UIView {
+    private lazy var loginEntryView: LoginEntryView = {
+        let loginEntryView = LoginEntryView()
+        loginEntryView.translatesAutoresizingMaskIntoConstraints = false
```
Comment (bstien) [inline Demo/Fullscreen/LoginEntry/LoginEntryViewDemoView.swift]: You could use the FinniversKit initializer instead for `UIView`'s. That saves you the annoying line `x.translatesAutoresizingMaskIntoConstraints = false` 🎉

I think it applies to every view we have in FK, except for `Label` and `Button` which has their own init.

I.e.:
```swift
let loginEntryView = LoginEntryView(withAutoLayout: true)
```

Comment (fespinoza) [reply Demo/Fullscreen/LoginEntry/LoginEntryViewDemoView.swift]: 👍

**Comment by fespinoza** [comment]
This is ready @bstien, can you review it? (as it was approved before the last changes)

File: Sources/Fullscreen/LoginEntryView/LoginEntryDialogueView.swift [hunk 2]
```diff
@@ -0,0 +1,129 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public class LoginEntryDialogueView: UIView {
```
Comment (bstien) [inline Sources/Fullscreen/LoginEntryView/LoginEntryDialogueView.swift]: Does this class need to be public?

Comment (fespinoza) [reply Sources/Fullscreen/LoginEntryView/LoginEntryDialogueView.swift]: I guess it doesn't really have to be. I can fix that

**Pull request merged**